package.json

```diff
@@ -6,13 +6,19 @@
     "dev": "turbo watch dev",
     "lint": "turbo lint",
     "lint:fix": "turbo lint:fix",
-    "ncu": "turbo ncu"
+    "ncu": "turbo ncu",
+    "test": "turbo run test",
+    "test:ui": "turbo run test:ui"
   },
   "devDependencies": {
     "@biomejs/biome": "1.9.4",
     "@repo/typescript-config": "*",
+    "@vitejs/plugin-react": "^4.4.1",
+    "@vitest/ui": "^3.1.2",
+    "jsdom": "^26.1.0",
     "npm-check-updates": "^17.1.15",
-    "turbo": "^2.4.4"
+    "turbo": "^2.4.4",
+    "vitest": "^3.1.2"
```

Comment:
The dependencies should not be duplicated in the root and the project. It should be just one of them, having `vitest` in root is fine.